src/style.css

```diff
@@ -0,0 +1,113 @@
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-weight: 700;
+}
+
+.big-search-form {
+  margin: 20px 0;
+}
+
+.big-search-container {
+
+  display: flex;
+  position: relative;
+  width: 100%;
+  height: 70px;
+
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.big-search {
+  border: none;

```

Comment:
```suggestion
  border: none;

```
Спробуй також стилізувати бордер для цього інпута. Можливо, саме через це й не проходять тести. Зараз у тебе немає border-radius на цих полях. А мав би бути:

<img width="1303" alt="Снимок экрана 2021-09-17 в 13 57 10" src="https://user-images.githubusercontent.com/66570142/133771988-00d8b384-0095-4ad1-b471-9dc007b294ae.png">

Reply:
Позбувся повторення коду та пофіксив border-radius, але тести досі не не проходять. Sadness...:(